content/events/2020/07/2020-07-28-2020-government-ux-summit.md

```diff
@@ -0,0 +1,119 @@
+
+{{< accordion kicker="Day 1 of the UX Summit" title="UX Methods - July 28" icon="fas fa-copy" >}}
+
+ **First Session – 10-minute presentations 11:30 AM – 12:30 PM ET**
+
+ - **Becoming a UX Houdini: Finding Opportunities in the Face of Constrictions** - *John Pull, Library of Congress*
+The Library of Congress’s Harry Houdini Collection provides insights and inspiration for UX professionals who feel chained down by the regulations, legacy systems, and PR challenges of government projects.
+
+ - **UX Research via Dot-Voting** - *Angela Smithers and Maya Rhodan, National Museum of African American History and Culture*
+A presentation on adapting the [discover method](https://methods.18f.gov/discover/dot-voting/) from 18F to perform quick user testing and get quantifiable results that stakeholders can comprehend.
+
+ - **Development of a Standardized Formula to Calculate Severity Scores in Multitask Usability Testing** - *Anthony J. Schulzetenberg, U.S. Census Bureau*
+Severity scores are often used in usability research to provide stakeholders a measure of urgency for the usability problems that were identified during testing. However, the methods used to develop these scores often lack rigor or applicability to multitask usability research. Because government UX research often involves testing multiple tasks, we propose an intuitive method to calculate severity scores for each usability problem that takes into account both the effectiveness and efficiency of the user, weighting effectiveness-related problems more heavily. We then can calculate an overall severity score and see each problem’s relative strain on the system’s total usability.
+
+ - **Solving the Lost Sock Problem: A Useful Method to Track Standards and UX Requirements in an Agile Software Development Process** - *Margo Kabel, Veterans Health Administration*
+This presentation proposes a useful method to track standards and UX requirements in an agile software development process. This presentation is also a case study of medication information standards implementation at the Veterans Health Administration.
+
+**Second Session 1:30 – 2:30 PM ET**
+
+ - **Ohana for Digital Service Design (or Baking Accessibility into your Digital Product Lifecycle)** - *Jennifer Strickland and Kendra Skeene, Ad Hoc (VA Modernization)*
+Designing and developing digital products that are accessible to all doesn’t just make good business sense — it’s also the law. Yet it’s often an overlooked part of the product development lifecycle. The best and most cost-effective way to ensure your product is accessible is to start with accessibility. In this session, participants will learn how to incorporate accessibility and inclusive design at each stage of the product lifecycle. From product requirements through user research, content strategy and copywriting, design, front-end development, and QA, consider accessibility using these effective guidelines. {{< /accordion >}}
```

Comment:
This one doesn't look like it's formatting correctly and maybe needs and extra space.